deny.toml

```diff
@@ -131,7 +131,7 @@ registries = [
 # https://embarkstudios.github.io/cargo-deny/checks/bans/cfg.html
 [bans]
 # Lint level for when multiple versions of the same crate are detected
-multiple-versions = "deny"
+multiple-versions = "warn"
```

Comment:
Instead of warn, can you add the duplicate dependency version to the list below? That's what we've been doing.